docs/sql-guide/functions/function-setexcludes.md

```diff
@@ -0,0 +1,58 @@
+---
+title: SETEXLCUDES()
+layout: default
+parent: Functions
+grand_parent: SQL guide
+---
+
+# SETEXCLUDES() function
+
+`SETEXCLUDES()` tests membership of a set of values within a set. It returns true if all of the members of `testset` do not exist in `targetset`
```

Comment:
"...a set of values within a set..."
First off, I'd say to state we're talking about IDSET/STRINGSET here because we don't really talk about "set" alone.
"`SETEXCLUDES()` tests if a specified value exists within an IDSET or STRINGSET data type column. 

I'd leave off "it returns true" because that's stated under the `## Results` heading.

I think in situations like this it's good to put a Before you begin to give links back to IDSET/STRINGSET

```
## Before you begin

* [IDSET data type](/docs/sql-guide/data-types/data-type-idset)
* [STRINGSET data type](/docs/sql-guide/data-types/data-type-stringset)
```

Another question: does this also work for SETQ data types? In which case this needs revisiting